src/App.js

```diff
@@ -8,29 +8,35 @@ import Feed from "./components/feed/Feed";
 import AllPropertiesData from "./components/allproperties/AllPropertiesData";
 import Profile from "./pages/profile/Profile";
 import Sales from "./components/sales/Sales";
-import Rent from "./components/rent/Rent"
-
-
-
+import Rent from "./components/rent/Rent";
+import PropertyDetails from "./components/propertydetails/PropertyDetails";
+import Description from "./components/propertydetails/Description";
+import Review from "./components/propertydetails/Review";
+import Features from "./components/propertydetails/Features";
 
 function App() {
   return (
     <div className="App">
       <Router>
         <Routes>
           <Route path="/" element={<Homepage />}></Route>
-          <Route path="/properties" element={<AllPropertiesData/>}></Route>
+          <Route path="/properties" element={<AllPropertiesData />}></Route>
           <Route path="/signup" element={<Signup />}></Route>
           <Route path="/login" element={<Login />}></Route>
           <Route path="/dashboard" element={<Dashboard />}></Route>
-          <Route path="/my-properties" element={<Feed/>}></Route>
-          <Route path="/post-a-property" element={<PropertiesInputData/>}></Route>
-          <Route path="/profile" element={<Profile/> }></Route>
-          <Route path="/sales" element={<Sales/>}></Route>
-          <Route path="/rent" element={<Rent/>}></Route>
-     
-      
-          
+          <Route path="/my-properties" element={<Feed />}></Route>
+          <Route
+            path="/post-a-property"
+            element={<PropertiesInputData />}></Route>
+          <Route path="/profile" element={<Profile />}></Route>
+          <Route path="/sales" element={<Sales />}></Route>
+          <Route path="/rent" element={<Rent />}></Route>
+
+          <Route path="/properties/:id" element={<PropertyDetails />}>
+            <Route path="features" element={<Features />} />
```

Comment:
Why is your path name not having slash"/" are you sure that will route?

Reply:
yes. According to a video i watched, i learnt using a slash on a route which is inside of another route is optional

Reply:
> Why is your path name not having slash"/" are you sure that will route?

yes. According to a video i watched, i learnt using a slash on a route which is inside of another route is optional